### ISSUE TITLE
Question about eval

### ISSUE THREAD

**Issue by renyujie-del** [xwjabc/hed#47]
您好，我在运行下面语句时有一些问题
cd eval
(echo "data_dir = '../output-mypretrain/test'"; cat eval_edge.m)|matlab -nodisplay -nodesktop -nosplash
这个语句怎么用？我在运行时直接复制您的这条语句运行？还是需要自己先创建一些文件？

**Comment by xwjabc** [comment]
Yes. You can directly run this command. There is no need to create any file in advance.

**Comment by renyujie-del** [comment]
Hello, I still have some questions. I just came into contact with image segmentation. I don't understand the overall idea and process. What have you done in image preprocessing? Is the image segmented according to the gray value of the image? Can you give me an overview of the whole segmentation process

**Comment by xwjabc** [comment]
Hmmmm actually HED is not an image segmentation (e.g., semantic segmentation/instance segmentation) framework. If you are looking for segmentation methods, you can take a look at DeepLab series (semantic segmentation) and Mask R-CNN (instance segmentation).
